core/src/main/java/org/apache/iceberg/rest/responses/CreateNamespaceResponse.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest.responses;
+
+import java.util.Map;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
+
+/**
+ * Represents a REST response for a request to create a namespace / database.
+ * <p>
+ * The properties returned should include all the user provided properties from the
+ * request, as well as any server-side added properties.
+ * <p>
+ * Example server-side added properties can be implementation specific, but might include
+ * such things as `created_at` or `owner`.
```

Comment:
The spec needs to be updated to match this. I agree with returning the namespace and actual properties so that the caller doesn't have to immediately send a GET request, but we should make sure the implementation and spec match.

Reply:
Opened a PR to update the spec: https://github.com/apache/iceberg/pull/4039

Reply:
This has been merged.